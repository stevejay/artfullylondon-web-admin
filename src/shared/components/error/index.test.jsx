import React from 'react'

import Error from './index'
import errorType from '_src/domain/types/error-type'

it('should render a 500 error correctly', () => {
  const wrapper = shallow(<Error />)
  expect(wrapper).toMatchSnapshot()
})

it('should render a 404 error correctly', () => {
  const wrapper = shallow(<Error type={errorType.NOT_FOUND} />)
  expect(wrapper).toMatchSnapshot()
})
