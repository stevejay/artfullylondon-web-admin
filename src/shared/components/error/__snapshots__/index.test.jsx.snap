// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render a 404 error correctly 1`] = `
<BasicSection
  className="src-shared-components-error-index__section"
>
  <h1
    className="src-shared-components-error-index__heading"
  >
    <FaThumbsODown
      className="src-shared-components-error-index__icon"
    />
    Sorry—we couldn't find that info
  </h1>
</BasicSection>
`;

exports[`should render a 500 error correctly 1`] = `
<BasicSection
  className="src-shared-components-error-index__section"
>
  <h1
    className="src-shared-components-error-index__heading"
  >
    <FaThumbsODown
      className="src-shared-components-error-index__icon"
    />
    Oops—something went wrong
  </h1>
</BasicSection>
`;
